experiments/20161122-pacbio/barcode-insert-primers.md

```diff
@@ -0,0 +1,15 @@
+## Primers for inserting barcode
+
+Since the ORF of PB2 takes up the majority of the sequence and since the UTR needs to be intact for polymerase function, the barcode will be inserted in the pCI backbone.
+
+**Option 1: Before PB2**
```

Comment:
Best to indicate where the first `N` of the N<sub>8</sub> will be inserted, for example: "25 n.t. upstream of PB2 start."